travis-ymls/cmd-shim.travis.yml

```diff
@@ -0,0 +1,30 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : cmd-shim
+# Source Repo         : https://github.com/ForbesLindesay/cmd-shim.git
+# Travis Job Link     : https://travis-ci.com/github/santosh653/cmd-shim
```

Comment:
looks like we have failing builds here?

Reply:
@seth-priya , there are failures on AMD64 windows distro, but the code is running fine on power.

Reply:
can we exclude the failing builds in that case?

Reply:
@seth-priya Done the needful, Can you please confirm?

Reply:
@seth-priya Can you do the needful?

Reply:
@Siddhesh-Ghadi Can you please Review?
Thanks !!

Reply:
@gururajrkatti @Siddhesh-Ghadi Request you to please review?